doit/cmd_resetdep.py

```diff
@@ -0,0 +1,50 @@
+from .cmd_base import DoitCmdBase, check_tasks_exist
+from .cmd_base import subtasks_iter
+
+
+class ResetDep(DoitCmdBase):
+    name = "reset-dep"
+    doc_purpose = "recompute the state of file dependencies"
+    doc_usage = "[TASK ...]"
+    doc_description = None
+
+    cmd_options = ()
+
+    def _execute(self, pos_args=None):
+        filter_tasks = pos_args
+
+        # dict of all tasks
+        tasks = dict([(t.name, t) for t in self.task_list])
+
+        if filter_tasks:
+            # list only tasks passed on command line
+            check_tasks_exist(tasks, filter_tasks)
+            # get task by name
+            task_list = []
+            for name in filter_tasks:
+                task = tasks[name]
+                task_list.append(task)
+                task_list.extend(subtasks_iter(tasks, task))
+        else:
+            task_list = self.task_list
+
+        write = self.outstream.write
+        for task in task_list:
+            try:
+                run_status = self.dep_manager.get_status(task, tasks)
+            except Exception:
+                run_status = 'run'
+
+            if run_status == 'up-to-date':
+                write("skip %s\n" % task.name)
+                continue
+
+            task.values = self.dep_manager.get_values(task.name)
+            try:
+                self.dep_manager.save_success(task)
+            except OSError:
+                write("skip (missing file_dep) %s\n" % task.name)
```

Comment:
You used the word `skip` for both:
  - unsuccessful operation (fail because there is no file)
  - success nothing to be done

Could we say? "failed - missing file_dep, can not set bla bla bla"